integration_tests/test_project_folder_and_packaged_src_are_equivalent.py

```diff
@@ -8,64 +8,78 @@
 
 
 def main(cli_args: List[str]):
-    # TODO: when possible, also add multiversx/mx-exchange-sc (as of May 2023, it references mx-sdk-rs < v0.41.0, thus cannot be used for testing reproducible builds v5).
-    project_path = download_project_repository("https://github.com/multiversx/mx-reproducible-contract-build-example-sc/archive/refs/tags/v0.4.0.zip", "mx-exchange-sc-main")
-    parent_output_using_project = PARENT_OUTPUT_FOLDER / "using-project"
-    parent_output_using_packaged_src = PARENT_OUTPUT_FOLDER / "using-packaged-src"
-
-    shutil.rmtree(parent_output_using_project, ignore_errors=True)
-    shutil.rmtree(parent_output_using_packaged_src, ignore_errors=True)
-
-    check_project_folder_and_packaged_src_are_equivalent(project_path, parent_output_using_project, parent_output_using_packaged_src, ["adder", "multisig"])
+    repository_url = "https://github.com/multiversx/mx-reproducible-contract-build-example-sc"
+    tag = "0.4.7-beta.1"
```

Comment:
Will be adjusted at a later time, after this merge, once a proper tag is available.

See: https://github.com/multiversx/mx-reproducible-contract-build-example-sc/pull/20